submissions/evgeniy24/memory-pair-game/styles.css

```diff
@@ -0,0 +1,79 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+}
+h1 {
+  margin-top: 10px;
+  text-align: center;
+}
+
+.game {
+  margin-left: auto;
+  margin-right: auto;
+  width: 800px;
+  height: 600px;
+  display: flex;
+  flex-wrap: wrap;
+  perspective: 1500px;
+}
+
+.card {
```

Comment:
Add `cusrsor: pointer;` to the cards on hover. It would improve UX of your app

Reply:
fixed! Pleasee see the result!

Reply:
I see it in your code, but don't see in your demo :(

Reply:
Fixed in Demo !